src/overlays/actors/ovl_En_Invadepoh/z_en_invadepoh.c

```diff
@@ -618,8 +618,8 @@ unkStruct80B50350 D_80B50350[10];
 EnInvadepoh* D_80B503F0;
 EnInvadepoh* D_80B503F4;
 EnInvadepoh* D_80B503F8;
-AnimatedMaterial* D_80B503FC;
-AnimatedMaterial* D_80B50400;
+AnimatedMaterial* alienEyeBeamTexAnim;
+AnimatedMaterial* alienEmptyTexAnim;
```

Comment:
```suggestion
AnimatedMaterial* sAlienEyeBeamTexAnim;
AnimatedMaterial* sAlienEmptyTexAnim;
```

Reply:
Yeah they probably should. In general `bss` should also be static since they are file specific, we just don't because IDO + bss is a nightmare, let alone static bss that wouldn't show up in the mapfile.

Reply:
Sure, that makes sense. Perhaps the style guide should be updated to mention that *all* file-specific variables should start with `s`.